docs/general/getting_started.md

```diff
@@ -15,6 +15,12 @@ And then execute:
 ```
 $ bundle
 ```
+### Available Commands
```

Comment:
I like that we add some rails-specific notes, but I thing they should rather be at the end of the file than at the top.